daemon/create.go

```diff
@@ -19,6 +19,7 @@ func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hos
 		return "", nil, fmt.Errorf("Config cannot be empty in order to create a container")
 	}
 
+	daemon.adaptContainerSettings(hostConfig)
```

Comment:
Yes this setting *2 is redundant if we don't have `memsw` support, but this warning will show up either we put this function before or after. And I thought it'll be safer if we can verify configs after adapt them, maybe it'll avoid some mistakes in the future when `adaptContainerSettings` grows up. Set memsw *2 then set it to -1 doesn't hurt anyone, WDYT?